nav2_bt_navigator/src/navigate_to_pose_behavior_tree.cpp

```diff
@@ -52,6 +53,9 @@ NavigateToPoseBehaviorTree::NavigateToPoseBehaviorTree()
   // Register our custom decorator nodes
   factory_.registerNodeType<nav2_tasks::RateController>("RateController");
 
+  // Register our custom Recovery nodes
```

Comment:
The Recovery node is a control node. Therefore the comment should be: 

`// Register our custom control nodes`

Reply:
Fixed.